Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs

```diff
@@ -10,7 +11,7 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos
     [AddComponentMenu("Scripts/MRTK/Examples/ChangeManipulation")]
     public class ChangeManipulation : MonoBehaviour
     {
-        public ManipulationHandler manipulatedObject;
+        public GameObject manipulatedObject;
```

Comment:
We should update these to either be [serialized private with public accessors](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation) or just public properties if serialization support isn't needed.